tests/functional/test_install.py

```diff
@@ -887,22 +887,6 @@ def test_install_package_with_target(script, with_wheel):
     result.did_update(singlemodule_py)
 
 
-@pytest.mark.parametrize("target_option", ['--target', '-t'])
-def test_install_package_to_usersite_with_target_must_fail(script,
```

Comment:
Why are you dropping this test? Again, it seems unrelated to the stated purpose of this PR.